devcontainer-mode.el

```diff
@@ -39,11 +39,18 @@ executable that often.")
       (expand-file-name (project-root proj))
     (user-error "Not in a project")))
 
+;; Spec: https://containers.dev/implementors/spec/#devcontainerjson
+(defun devcontainer-config-files ()
+  "Get the JSON config files for the current project."
+  (let ((default-directory (devcontainer--root)))
+    (append (seq-filter #'file-exists-p '(".devcontainer/devcontainer.json" ".devcontainer.json"))
+            (file-expand-wildcards ".devcontainer/*/devcontainer.json"))))
```

Comment:
When I remove the `(file-expand-wildcards ".devcontainer/*/devcontainer.json")` call, all the tests still pass.

Reply:
Yep, you're right, I've pushed a fix!